pkg/gadgets/profile/block-io/tracer/tracer.go

```diff
@@ -17,24 +17,21 @@
 
 package tracer
 
-// #include <linux/types.h>
-// #include "./bpf/biolatency.h"
-import "C"
-
 import (
 	"encoding/json"
 	"fmt"
 	"unsafe"
 
 	"github.com/cilium/ebpf"
 	"github.com/cilium/ebpf/link"
+	"github.com/moby/moby/pkg/parsers/kernel"
+
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/types"
-	"github.com/moby/moby/pkg/parsers/kernel"
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET -cc clang biolatency ./bpf/biolatency.bpf.c -- -I./bpf/ -I../../../../${TARGET}
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET -cc clang biolatencyBefore ./bpf/biolatency.bpf.c -- -I./bpf/ -I../../../../${TARGET} -DKERNEL_BEFORE_5_11
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET -type hist -type hist_key -cc clang biolatency ./bpf/biolatency.bpf.c -- -I./bpf/ -I../../../../${TARGET}
```

Comment:
They are actually already there: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadgets/profile/block-io/tracer/biolatency_bpfel_arm64.go#L16

Looks like it's generating it here because we don't use `-no-global-types` for bpf2go in this case and the struct is used inside the maps. I still think it's better to explicitly mention them for bpf2go here, but it seems like we could omit it in some cases.